deploy/helm/sumologic/values.yaml

```diff
@@ -28,6 +28,24 @@ eventsDeployment:
       memory: 256Mi
       cpu: "100m"
 
+otelcolDeployment:
+  nodeSelector: {}
+  tolerations: {}
+  replicas: 1
+  resources:
+    limits:
+      memory: 2Gi
+      cpu: 1
+    requests:
+      memory: 384Mi
+      cpu: "200m"
+  # Memory Ballast size should be max 1/3 to 1/2 of memory.
+  memBallastSizeMib: "683"
+  image:
+    name: "omnition/opentelemetry-collector-contrib"
+    tag: "0.0.5"
+    pullPolicy: "Always"
```

Comment:
Any reason to not use `IfNotPresent`?

Reply:
We were thinking about the `latest` anyway until we have a real release to speed up the prototyping. But we can always just use helm parameters for that. Let me change that.